sensor/common/deploymentenhancer/component_test.go

```diff
@@ -82,6 +82,34 @@ func (s *ComponentTestSuite) TestExtractAndEnrichDeployments() {
 	s.Len(actual, expected)
 }
 
+func (s *ComponentTestSuite) TestEnhanceDeploymentsEmptyMessage() {
+	dQueue := make(chan *central.DeploymentEnhancementRequest, 10)
+	de := DeploymentEnhancer{
+		responsesC:       make(chan *message.ExpiringMessage),
+		deploymentsQueue: dQueue,
+		storeProvider:    s.mockStoreProvider,
+	}
+	msg := &central.DeploymentEnhancementRequest{}
+
+	actual := de.enhanceDeployments(msg)
+
+	s.Empty(actual)
+}
+
+func (s *ComponentTestSuite) TestEnhanceDeploymentsNoDeployments() {
+	dQueue := make(chan *central.DeploymentEnhancementRequest, 10)
+	de := DeploymentEnhancer{
+		responsesC:       make(chan *message.ExpiringMessage),
+		deploymentsQueue: dQueue,
+		storeProvider:    s.mockStoreProvider,
+	}
+	msg := &central.DeploymentEnhancementRequest{Msg: &central.DeploymentEnhancementMessage{Id: uuid.NewV4().String()}}
+
+	actual := de.enhanceDeployments(msg)
+
+	s.Empty(actual)
```

Comment:
Same here - we use `msg` and `actual` only once, so there is no need to assign them to vars

Reply:
These are used once and could be merged, correct.
I try to follow the [AAA pattern](https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/) (Arrange, Act, Assert) pattern in my tests, as I found this makes for nicely readable code.

Given that Go breaks with some conventions, should I collapse this into one call, or should I keep using the pattern?

Reply:
I didn't even know that such thing is called a pattern :) 
The biggest challenge in patterns is to know when to use them and when not. I feel that this example is so simple that there is no need to apply it. However, if the rest of this big file follows that pattern (I haven't checked it), then we can keep it.